docs/usage.md

```diff
@@ -7,14 +7,16 @@ The easiest way to get started with enquirer is to pass a [question object](#pro
 ```js
 const { prompt } = require('enquirer');
 
-const response = await prompt({
-  type: 'input',
-  name: 'username',
-  message: 'What is your username?'
-});
+async function start() {
```

Comment:
What's the reason for this change? Earlier in the readme it states that awaited prompts need to be in async functions.

Reply:
Yes, but I thought having a complete example might be good, where a user can just copy paste everything & run it.